OloEngine/src/OloEngine/Renderer/Commands/FrameDataBuffer.cpp

```
@@ -160,4 +177,237 @@ namespace OloEngine
+                scratch.transforms.size() * 2,
+                static_cast<sizet>(scratch.transformCount + count));
+            scratch.transforms.resize(newCapacity);
+        }
+
+        u32 localOffset = scratch.transformCount;
+        scratch.transformCount += count;
+
+        return localOffset;
+    }
+
+    void FrameDataBuffer::WriteBoneMatricesParallel(u32 workerIndex, u32 localOffset,
```

Clang (14.0.6)

[warning] method 'WriteBoneMatricesParallel' can be made static

(readability-convert-member-functions-to-static)